### DISCUSSION TITLE
bossbarmsg Names

### DISCUSSION THREAD

**Discussion by ghost** [Zrips/CMI#6000] [Q&A]
If one forgot the name of a bossbarmsg, how does one list them, or remove all bossbarmsg from everyone on the server?

(UPDATE): **No longer needed** I dont know how to remove this post...

**Comment by mrfloris** [comment]
I am curious about this though;
```
> bossbar list
[09:23:25 INFO]: There are no custom bossbars active
>
```
Would that do the trick?
